kotlinx-coroutines-test/src/TestCoroutineDispatcher.kt

```diff
@@ -128,14 +128,15 @@ public class TestCoroutineDispatcher: CoroutineDispatcher(), Delay, DelayControl
         }
 
     // The ordered queue for the runnable tasks.
-    private val queue = ThreadSafeHeap<TimedRunnable>()
+    private val queue = PriorityBlockingQueue<TimedRunnable>()
 
     // The per-scheduler global order counter.
-    private val _counter = atomic(0L)
+    private val counter = atomic(0L)
```

Comment:
I've named them with underscore for consistency with `_state` and `_decision` variables in `CancellableContinuationImpl`, because `_counter.value` looks subjectively better than `counter.value`. See also disucssion on this style here: https://github.com/Kotlin/kotlinx.coroutines/pull/890#issuecomment-485159899